src/Frost/GitPlugin.hs

```diff
@@ -0,0 +1,14 @@
+module Frost.GitPlugin where
```

Comment:
Should be called more something like `ContributorsGitPlugin`

Reply:
`GitContributorsPlugin`